nautobot_netbox_importer/tests/fixtures/3.7/summary.json

```diff
@@ -8669,6 +9338,18 @@
                     "default_value": null,
                     "disable_reason": ""
                 },
+                "object_types": {
```

Comment:
How does this differ from the `content_types` field?

Reply:
Seems to be the same, mapped